lib/egon/overcloud/undercloud_handle.rb

```diff
@@ -39,6 +39,10 @@ def execute_workflow(workflow, input, wait_for_complete = true)
       workflow_execution_id
     end
 
+    def workflow_action_execution(action_name, params = {})
+      return JSON.parse(service('Workflow').create_action_execution(action_name, params).body["output"])["result"]
```

Comment:
I'm seeing:/home/rwsu/.rvm/gems/ruby-1.9.3-p545/gems/egon-1.1.0/lib/egon/overcloud/undercloud_handle.rb:43:in `workflow_action_execution': uninitialized constant Overcloud::UndercloudHandle::JSON (NameError)

I think it is missing require 'json'.